### ISSUE TITLE
Model quality is not same good as tower 8 GPU

### ISSUE THREAD

**Issue by weixsong** [uber/horovod#115]
Hi,

Thanks very much for this great work of Horovod. It's really made the distributed TF more easier.

I have tried 16 GPU and 24 GPU, the speedup ratio is nearly linear.

But the distributed model is not the same good as my Tower 8 GPU way. (In tower implementation, gradients are averaged and then do a parameters update step.). 
Regarding the distributed model, training steps is more than 8 GPU tower way, so it is supposed to be better than 8 GPU model, but actually not.

Is there any trick that what hyper parameters that  I need to adjust according to distributed GPU number or something?

Regards,
Wei

**Comment by alsrgv** [comment]
@weixsong, typically you'd scale your learning rate with the # of GPUs.  As you increase # of GPUs, it may be helpful to also do learning rate warmup, like in this [example model](https://github.com/uber/horovod/blob/master/examples/keras_mnist_advanced.py).

**Comment by weixsong** [comment]
@alsrgv , thanks.
